KeywordSearch/src/org/sleuthkit/autopsy/keywordsearch/Keyword.java

```diff
@@ -57,7 +58,7 @@
      *                    than a substring.
      */
     Keyword(String searchTerm, boolean isLiteral, boolean isWholeWord) {
```

Comment:
Can you have this constructor call the 5 parameter one instead of it assigning the values?